facts/test_datasources.py

```diff
@@ -96,3 +97,25 @@ def test_switchtypes_tsv():
     }
     result, err = ds.test_tsvfile(meta)
     assert result, err
+
+
+def test_vlansd_tsv():
+    """test vlans.d/"""
+
+    vlansddir = "../switch-configuration/config/vlans.d/"
+    for filename in os.listdir(vlansddir):
+        meta = {
+            "file": vlansddir + filename,
+            "header": False,
+            "count": "6+",
+            "cols": [
+                ds.isuntested,
```

Comment:
Can add these types of tests for the columns but the main concern with this was the number of cols and detecting any kind of issues with the tab delimiting

Reply:
You do need to make a provision for blank lines. Currently the script is failing on the blank line at the end of vlans.d/Expo